physics/sfc_drv_ruc.meta

```diff
@@ -729,6 +729,24 @@
   type = logical
   intent = in
   optional = F
+[xlat_d]
+  standard_name = latitude_in_degree
+  long_name = latitude in degree north
+  units = degree_north
+  dimensions = (horizontal_loop_extent)
+  type = real
+  kind = kind_phys
+  intent = in
+  optional = F
+[xlon_d]
+  standard_name = longitude_in_degree
+  long_name = longitude in degree east
+  units = degree_east
+  dimensions = (horizontal_dimension)
```

Comment:
@tanyasmirnova all the debug runs in `rt_ccpp_dev.conf` that use RUC LSM crashed with
```
138: An error occurred in ccpp_physics_run for group physics, block    1 and thread    1 (ntX=   1)
138: Detected size mismatch for variable GFS_Data(cdata%blk_no)%Grid%xlon_d in group physics before lsm_ruc_run, expected      384 but got       32
```
I think this is because line 745 says `(horizontal_dimension)` instead of `(horizontal_loop_extent)`.

Can you please fix this and run at least one manual test in DEBUG mode?

Thanks!